SU-HSDO/suhumsci
# SHS-5093: A11y Level A: Element IDs are not unique, on search results page

# READY FOR REVIEW

## Summary
- Create unique id for each instance of the default search 

## Steps to Test
Do the following for several sites:
1. Navigate to `/search`
2. Verify search form element ids are unique

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
